include/oneapi/dpl/experimental/kt/internal/esimd_radix_sort_kernels.h

```diff
@@ -27,7 +27,7 @@ namespace oneapi::dpl::experimental::kt::esimd::__impl
 
 template <bool __is_ascending, ::std::uint8_t __radix_bits, ::std::uint16_t __data_per_work_item,
           ::std::uint16_t __work_group_size, typename _KeyT, typename _InputT>
-void
+inline __attribute__((always_inline)) void
```

Comment:
Could you please pack this part `inline __attribute__((always_inline)`
into some new define?

Reply:
Done